sdk/python/tests/test_cli_local.py

```diff
@@ -1,18 +1,22 @@
+import json
 import tempfile
 from contextlib import contextmanager
 from pathlib import Path
 from textwrap import dedent
 
 import assertpy
 import pytest
+from google.protobuf.json_format import MessageToDict
 
 from feast.feature_store import FeatureStore
+from feast.protos.feast.core.Registry_pb2 import Registry as RegistryProto
 from tests.cli_utils import CliRunner
 from tests.online_read_write_test import basic_rw_test
 
 
 @pytest.mark.integration
-def test_workflow() -> None:
+@pytest.mark.parametrize("old_registry_config", [False, True])
```

Comment:
Instead of having a `True`, `False`, why not have the config as the parameter?

Reply:
I didn't know if it was possible since it requires `data_path`, but if you think it's cleaner, I can use the `"...{}...".format(str)` notation and have it be it be pytest param. Also, I'm assuming you mean just the `registry` section of the config and not the whole config.